webapp/app/js/config/sample.local.conf.js

```diff
@@ -3,17 +3,19 @@
  * file, you can obtain one at http://mozilla.org/MPL/2.0/. */
 
 'use strict';
+var production = "https://treeherder.mozilla.org";
+var stage = "https://treeherder.allizom.org";
+var dev = "http://treeherder-dev.allizom.org";
+var same_host = "";
```

Comment:
The name is arbitrarily picked, and I originally used 'local', however it occurred to me that when Vagrant is used, it's not actually connecting to localhost, it's connecting to 192.168.33.10 via the hosts local.treeherder.mozilla.org entry. I didn't want to call it 'vagrant' since the empty string is also used for stage/prod. So same_host seemed more appropriate. Happy for suggestions though :-)